ctoec/data-collection
# Automated Monthly Emails

## Background
This PR sets up automated monthly reminder emails to go out to all providers on the first of each month, reminding them to update their roster.

## GitHub Issue
https://github.com/ctoec/data-collection/issues/1357

## Associated PRs
- https://github.com/skylight-hq/ctoec-devops/pull/220

## Validation Plan
- Start up the app
- Change the target email address, name, and time signature to match the validator and the day (e.g. to test whether you can receive an email scheduled for 2:35 PM on September 10th, you would use `35 14 10 9 *` as the cron signature in the first param of `schedule`).
- Wait for the time to pass
- Check inbox for email

## Automated Testing
- [ ] If applicable, unit tests have been added to cover this changeset.

## Additional Context
Before merging this PR in, we need to change two things:
1.  The actual provider emails need to be put into the  `TO` field for the outgoing emails, and
2. The time signature of the cron job scheduler needs to be set to the start of the day on the first of each month (`0 0 1 * *`).
These aren't set right now because obviously we don't want to bombard providers with dummy emails while testing.
